redisinsight/ui/src/components/new-index/create-index-step/field-box/FieldTag.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Badge } from '@redis-ui/components'
+import {
+  FIELD_TYPE_OPTIONS,
+  FieldTypes,
+} from 'uiSrc/pages/browser/components/create-redisearch-index/constants'
+
+// TODO: Add colors mapping for tags when @redis-ui/components v38.6.0 is released
```

Comment:
`Badge` component in @redis-ui/components v38.6.0 supports custom color out of the box ([docs](https://redislabsdev.github.io/redis-ui/?path=/story/components-badge--with-custom-color)), so we can colorize the different types of fields, once we update the dependency version.

Note: Currently `@redis-ui/components` and `@redis-ui/styles`. Unfortunately, the latter comes with some breaking changes in its latest version, so the upgrade process might need additional effort, and I won't do it as part of this task.